docs/source/handling_advanced_stypes/handle_text.rst

```diff
@@ -1,38 +1,54 @@
 Handling Text Columns
-================================
+=====================
 
-:pyf:`PyTorch Frame` supports text columns by utilizing external pre-trained
+:pyf:`PyTorch Frame` supports text columns by utilizing or finetuning external pre-trained
 text embedding models, such as language models. Currently, we support
-:class:`stype.text_embedded<torch_frame.stype>` for text columns,
-which use a text embedding model to pre-encode text columns into embeddings
-(frozen during model training). We plan to support fine-tuning of the text embedding
-models soon.
+:class:`stype.text_embedded<torch_frame.stype>` and :class:`stype.text_tokenized<torch_frame.stype>`
+for text columns.
+:class:`stype.text_embedded<torch_frame.stype>` uses text embedding model(s) to pre-encode text columns into embeddings
+(those models are frozen during model training).
+:class:`stype.text_tokenized<torch_frame.stype>` finetunes underlying text embedding models by using the
+gradients backpropagated from the tabular learning. Now :pyf:`PyTorch Frame` pipeline supports supervised finetuning
+on text models with other libraries such as the `PEFT <https://huggingface.co/docs/peft/>`_.
 
 
 .. contents::
     :local:
 
 Handling Text Columns in a Benchmark Dataset
-----------------------------------------------
+--------------------------------------------
 
 :pyf:`PyTorch Frame` provides a collection of tabular benchmark datasets
 with text columns, such as :obj:`~torch_frame.datasets.MultimodalTextBenchmark`.
 
 In :pyf:`PyTorch Frame`, you can specify text columns as
 :class:`stype.text_embedded<torch_frame.stype>`. This will
-encode text columns using a user-specified text embedding model during the
+encode text columns using a user-specified text embedding model(s) during the
 dataset materialization stage.
+You can also specify text columns as
+:class:`stype.text_tokenized<torch_frame.stype>`, which will finetune
+user-specified text model(s) during the training stage.
```

Comment:
you can specify the stype of text columns in two ways.
begin{itemize}
1. stype.text_embedded: explain
2. stype.text_tokenized: explain